### PR TITLE
Add a lang attribute to brand component

### LINKED ISSUE #607: Add lang attribute to the brand component
**Is your feature request related to a problem? Please describe.**
We currently don't have a lang attribute defined for the Brand Component which would help screen readers to announce brand name in the correct accent. i.e `BBC News’ is announced with an English accent.

**Describe the solution you'd like**
Add a `lang` attribute to the `Brand` component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### LINKED ISSUE #1867: Add lang attribute to the brand component
**Is your feature request related to a problem? Please describe.**
We currently don't have a lang attribute defined for the Brand Component which would help screen readers to announce brand name in the correct accent. i.e `BBC News’ is announced with an English accent.

**Describe the solution you'd like**
Add a `lang` attribute to the `Brand` component.

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Resolves #1867 

**Overall change:** _Adds a lang attribute ie "en-gb" to ensure brand name ie `BBC news` is read with the correct accents"_

**Code changes:**

-_Wrap "BBC news" text inside a span with a lang attribute with the value set to "en_GB"_

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/containers/Brand/index.jsx [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
 const BrandContainer = () => {
   const { brandName, brandSVG } = useContext(ServiceContext);
+  const defaultBrandName = 'BBC News';
+  let localisedBrandName;
 
-  return <Brand brandName={brandName} svg={brandSVG} />;
+  if (brandName !== defaultBrandName) {
+    localisedBrandName = (
+      <Fragment>
+        <span lang="en_GB">{defaultBrandName}</span>
```
Comment (ghost) [inline src/app/containers/Brand/index.jsx:13]:
```suggestion
        <span lang="en-GB">{defaultBrandName}</span>
```

File: src/app/containers/Brand/index.jsx [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+        {brandName.substring(
+          brandName.indexOf(defaultBrandName) + defaultBrandName.length,
+        )}
+      </Fragment>
+    );
+  }
+
+  return <Brand brandName={localisedBrandName || brandName} svg={brandSVG} />;
```
Comment (ghost) [inline src/app/containers/Brand/index.jsx:21]: `Brand` expects `brandName` to be a string but `localizedBrandName` is returning a `node`

**Review by ghost — CHANGES_REQUESTED** [review]
I think it would be better for this work to be done on the `psammead` repo.

**Comment by EinsteinNjoroge** [comment]
@sadickisaac I'm gonna close this PR and move this implementation to the psammead repo.

**Pull request closed without merging**